install.sh

```diff
@@ -3,11 +3,11 @@
 yum -y update
 sudo yum install -y yum-utils
 sudo yum install -y centos-release-scl
-sudo yum-config-manager --enable rhel-server-rhscl-7.2-rpms
-sudo yum install -y devtoolset-7.2
+sudo yum-config-manager --enable rhel-server-rhscl-7.2*-rpms
```

Comment:
does this work on local?

Reply:
i checked now its not working

Reply:
i check with the star its not working . i rollback to the previous version

Reply:
we should not be sending prs like this that we know will fail for sure.